test/d2l-localize-behavior.html

```diff
@@ -293,73 +293,6 @@
 
 	});
 
-	describe('data/time/number overrides', function() {
```

Comment:
These tests should have been failing since the latest updates to Intl, which don't allow overriding of time formats anymore.